src/main.c

```diff
@@ -177,24 +206,16 @@ main(int argc, char *argv[])
 
 	menu_init(&server);
 
```

Comment:
instead of the change below, would 

(void)wl_event_loop_dispatch(server.wl_event_loop, 0);

work ?

Reply:
Potentially that could work (although I didn't look into it in detail). However, I think we should not interfere with eventloop internals here.